hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/ContainerDiscrepancyInfo.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.recon.api.types;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.hadoop.hdds.scm.pipeline.Pipeline;
+
+import java.util.List;
+
+/**
+ * Metadata object that represents a Container Discrepancy Info.
+ */
+public class ContainerDiscrepancyInfo {
+
+  @JsonProperty("containerId")
+  private long containerID;
+
+  @JsonProperty("omContainerState")
+  @JsonInclude(JsonInclude.Include.NON_EMPTY)
+  private long omContainerState;
+
+  @JsonProperty("scmContainerState")
+  @JsonInclude(JsonInclude.Include.NON_EMPTY)
+  private long scmContainerState;
```

Comment:
It looks like scmContainerState and omContainerState are not used in anywhere.

Reply:
> It looks like scmContainerState and omContainerState are not used in anywhere.

Thanks for pointing out Sammi, I have removed them, earlier I was thinking to use, but I don't have a mechanism to for states of mismatched containers. Pls review again.